RELEASE/scripts/autoscend/combat/auto_combat_default_stage2.ash

```diff
@@ -362,6 +362,13 @@ string auto_combatDefaultStage2(int round, monster enemy, string text)
 			return useSkill($skill[lightning strike]);
 		}
 
+		if(canUse($skill[Darts: Aim for the Bullseye]) && have_effect($effect[Everything Looks Red]) == 0 && (wantFreeKillNowEspecially || !reserveFreekills))
```

Comment:
We need to move the Bullseyee code up to line 355, so it ignores any attempts to save the free runs for the desert. These can't be saved for later.

At the moment, this can block out using any Bullseyes while the desert is unlocked but not completed.

Reply:
I can also see this being blocked if we have CMC and waiting on doing the Cyrpt. I think I can probably just remove the last conditional and it should use it as soon as it can.